src/wp-includes/widgets.php

```diff
@@ -1457,22 +1457,22 @@ function wp_map_sidebars_widgets( $existing_sidebars_widgets ) {
 }
 
 /**
- * Compares a list of sidebars with their widgets against a whitelist.
+ * Compares a list of sidebars with their widgets against an allowed list.
  *
  * @since 4.9.0
  *
- * @param array $sidebars_widgets List of sidebars and their widget instance IDs.
- * @param array $whitelist        Optional. List of widget IDs to compare against. Default: Registered widgets.
- * @return array Sidebars with whitelisted widgets.
+ * @param array $sidebars_widgets  List of sidebars and their widget instance IDs.
```

Comment:
Is the alignment right here? Sometimes GH diffs are weird.

Reply:
It is not, good catch!